src/sentry/conf/urls.py

```diff
@@ -45,9 +45,15 @@ def handler500(request):
     return HttpResponseServerError(t.render(Context(context)))
 
 
+def handler_healthcheck(request):
+    from django.http import HttpResponse
+    return HttpResponse('ok')
+
+
 urlpatterns = patterns('',
     url(r'^admin/', include(admin.site.urls)),
     url(r'^500/', handler500, name='error-500'),
     url(r'^404/', handler404, name='error-400'),
+    url(r'^healthcheck/', handler_healthcheck, name='healthcheck'),
```

Comment:
Can you name this something like `_health` instead? That way we avoid any more conflicts with potential organization names, etc.

Also, you'll want a trailing `$`.